Makefile

```diff
@@ -220,15 +220,11 @@ clean:
 	@echo "cargo clean in root directory"
 	@cargo clean
 
-.PHONY: update
-update:
-	@echo "Running cargo update.."
-	@cd enclave-runtime && cargo update
-	@cd enclave-runtime && cargo update -p sp-std --precise f651d45ce5742bc60fe8ae518c035d1638ae83d2
-	@cd enclave-runtime && cargo update -p sgx_tstd --precise 7c07ce0bfbacd3f4f2af53a2cdef9539018be73c
-	@cargo update
-	@cargo update -p sp-std --precise f651d45ce5742bc60fe8ae518c035d1638ae83d2
-	@cargo update -p sgx_tstd --precise 7c07ce0bfbacd3f4f2af53a2cdef9539018be73c
+.PHONY: pin-sgx
+pin-sgx:
+	@echo "Pin sgx dependencies to 6cf73140d5e9b05366a2ae4f8fe8141449e7204f"
+	@cd enclave-runtime && cargo update -p sgx_tstd --precise 6cf73140d5e9b05366a2ae4f8fe8141449e7204f
+	@cargo update -p sgx_tstd --precise 6cf73140d5e9b05366a2ae4f8fe8141449e7204f
```

Comment:
So we don't need the `cargo update -p sp-std` anymore in this command?

Reply:
If an update is necessary, we need to run cargo update -p sp-std. But it breaks so very often (it's broken now btw as well due to substrate moving some crates..). But I agree with @clangenb reasoning, that having this command in the makefile suggests a working "update", which is just not the case.

Reply:
Okay I see, thanks for the clarification 👍